common.yml

```diff
@@ -727,6 +727,7 @@ functions:
           GOROOT="" set_goenv || exit
           export EVG_PLATFORM=${_platform}
           go run release/release.go build-archive
+          go run release/release.go build-msi
```

Comment:
I just realized; we should probably add `set -o errexit` to the beginning of the command here, so that this actually fails if `build-archive` fails

Reply:
sgtm